netcore/Makefile.am

```diff
@@ -53,9 +53,10 @@ clean:
 # '-parallel none -verbose' for debug, `-parallel all` to run all tests in parallel
 # we need COREFX_ROOT only for xunit.console.deps.json and CoreFx.Private.TestUtilities.dll
 xtest-%: prepare check-env dl-test-assets
+	echo -n "***************** $* *********************"
 	ln -sf $(CURDIR)/sdk/$(NETCORESDK_VERSION)/System.Text.Encoding.CodePages.dll assets/extracted/$*/
 	cp $(COREFX_BINDIR)/runtime/netcoreapp-OSX-Debug-x64/xunit.console.deps.json assets/extracted/$*/
-	cp $(COREFX_BINDIR)/runtime/netcoreapp-OSX-Debug-x64/CoreFx.Private.TestUtilities.dll assets/extracted/$*/
+	cp $(COREFX_BINDIR)/runtime/netcoreapp-OSX-Debug-x64/{CoreFx.Private.TestUtilities.dll,Microsoft.DotNet.PlatformAbstractions.dll} assets/extracted/$*/
```

Comment:
Why was this necessary? My CoreFX build doesn't have this file. It does exist in the SDK though: `./sdk/3.0.100-preview4-010759/Microsoft.DotNet.PlatformAbstractions.dll`.

Reply:
Most test suites seem to require it.

Reply:
I run `xunitall` without it and didn't hit it yet, so there must be some difference between our setups.

Reply:
Without it, I get:
     System.TypeInitializationException : The type initializer for 'System.PlatformDetection' threw an exception.
      ---- System.IO.FileNotFoundException : Could not load file or assembly 'Microsoft.DotNet.PlatformAbstractions, Version=2.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' or one of its dependencies.

Reply:
Do you have some specific xunit-<something> test where you can reproduce it? I will try it again, locally. Also, what CoreFX version do you have in COREFX_ROOT?

Reply:
yeah works for me without it, I suspect you need to update your local corefx repo
see https://github.com/dotnet/corefx/issues/28620 (was fixed a month ago)